src/pages/workspaces/workspace/Data.js

```diff
@@ -793,21 +794,6 @@ const WorkspaceData = _.flow(
 
     const toSortedPairs = _.flow(_.toPairs, _.sortBy(_.first));
 
-    const getRunningImportJobs = async () => {
-      try {
-        const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true);
-        const currentJobIds = _.map('jobId', asyncImportJobStore.get());
-        _.forEach((job) => {
-          const jobStatus = _.lowerCase(job.status);
-          if (!_.includes(jobStatus, ['success', 'error', 'done']) && !_.includes(job.jobId, currentJobIds)) {
-            asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId: job.jobId }));
-          }
-        }, runningJobs);
```

Comment:
This was doing more work checking statuses than it needed to, since the API request only returns running jobs.